article/presentation_article.md

```diff
@@ -0,0 +1,1287 @@
+# Title: GNOME Terminal automation on Fedora 38 with Wayland.
+
+# Table of contents
+- [Title: GNOME Terminal automation on Fedora 38 with Wayland.](#title-gnome-terminal-automation-on-fedora-38-with-wayland)
+- [Table of contents](#table-of-contents)
+- [Introduction](#introduction)
+- [Automation stack for GNOME Applications](#automation-stack-for-gnome-applications)
+  - [What are we using? Automation API - dogtail](#what-are-we-using-automation-api---dogtail)
+  - [How are we dealing with automation on Wayland? The gnome-ponytail-daemon](#how-are-we-dealing-with-automation-on-wayland-the-gnome-ponytail-daemon)
+  - [Giving the API a structure to be used in automation - behave](#giving-the-api-a-structure-to-be-used-in-automation---behave)
+  - [The automation suite result page in form of behave-html-pretty-formatter project](#the-automation-suite-result-page-in-form-of-behave-html-pretty-formatter-project)
+  - [Filling all the gaps and providing useful tools with the qecore project](#filling-all-the-gaps-and-providing-useful-tools-with-the-qecore-project)
+- [Full project example.](#full-project-example)
+  - [Basic machine setup required before any action.](#basic-machine-setup-required-before-any-action)
+  - [Installing, building and execution.](#installing-building-and-execution)
+  - [Errors you can encounter.](#errors-you-can-encounter)
+  - [The main queries you will be using.](#the-main-queries-you-will-be-using)
+  - [Explaining the rest of the gnome-terminal project.](#explaining-the-rest-of-the-gnome-terminal-project)
+- [Examples](#examples)
+- [Comparison of OpenQA vs Accessibility](#comparison-of-openqa-vs-accessibility)
+- [Usage with GTK4](#usage-with-gtk4)
+- [Usage with Fedora 39](#usage-with-fedora-39)
+- [Reason for this article.](#reason-for-this-article)
+- [Future Plans/Aspirations -\> GNOMEAutomation](#future-plansaspirations---gnomeautomation)
+  - [Qecore](#qecore)
+  - [Accessibility](#accessibility)
+  - [The behave](#the-behave)
+  - [The dogtail API](#the-dogtail-api)
+  - [The gnome-ponytail-daemon](#the-gnome-ponytail-daemon)
+  - [The behave-html-pretty-formatter](#the-behave-html-pretty-formatter)
+- [Finally](#finally)
+- [Keywords](#keywords)
+- [Sources](#sources)
+
+
+# Introduction
+
+Hello and welcome.
+
+My name is Michal and I'm a Quality Engineer in Red Hat. I've been in Red Hat for over 7 years now and my responsibility is to write, maintain and improve an automation stack for GNOME Applications on RHEL while also having a responsibility for specific components and their automation suites, like `'gnome-control-center'`, `'gnome-terminal'`, `'gnome-shell'`, `'evolution'` and more.
```

Comment:
Escapes are used inconsistently, I would suggest `string` instead of `'string'` nearly everywhere - the only exception is if `'string'` is really a string (e.g. suppose variable `x` contains `'y'`)